integrationTests/consensus/testInitializer.go

```diff
@@ -91,7 +92,8 @@ func genValidatorsFromPubKeys(pubKeysMap map[uint32][]string) map[uint32][]shard
 	for shardId, shardNodesPks := range pubKeysMap {
 		shardValidators := make([]sharding.Validator, 0)
 		for i := 0; i < len(shardNodesPks); i++ {
-			v, _ := sharding.NewValidator(big.NewInt(0), 1, []byte(shardNodesPks[i]), []byte(shardNodesPks[i]))
+			v, _ := sharding.NewValidator(big.NewInt(0), 1, []byte(shardNodesPks[i]),
+				[]byte("addr_"+strconv.Itoa(int(shardId))+"_"+strconv.Itoa(i)))
```

Comment:
can you use sprintf instead? then you don't need all the string conversions and it's more readable.

Reply:
Done